app/user_agent.go

```diff
@@ -120,13 +128,16 @@ var browserNames = map[uasurfer.BrowserName]string{
 func getBrowserName(ua *uasurfer.UserAgent, userAgentString string) string {
 	browser := ua.Browser.Name
 
-	if strings.Contains(userAgentString, "Mattermost") {
```

Comment:
Here I prefer a chain of "ifs" (not if elses, only ifs with return each one). The case in this case is more confusing (at least for me). Mainly because the clauses aren't homogeneous.

Reply:
done